index.test.js

```diff
@@ -0,0 +1,98 @@
+const skeletorLocalServer = require('./index');
+
+jest.mock('express');
+
+global.console = {
+    log: jest.fn()
+};
+
+const logger = {
+    info: () => {},
+    error: () => {}
+};
+const options = {logger};
+let consoleSpy;
+
+describe('local server plugin', () => {
+
+    beforeEach(() => {
+        consoleSpy = jest.spyOn(global.console, 'log');
+    });
+
+    afterEach(() => {
+        consoleSpy.mockReset();
+    });
+
+    it('should error when no entry point specified', async () => {
+        const expectedError = {
+            status: 'error',
+            message: 'Error with config. Directory: testDir, Entry: undefined'
+        };
+        const config = {
+            currentDirectory: 'testDir'
+        };
+        await expect(skeletorLocalServer().run(config, options)).rejects.toEqual(expectedError);
+    });
+
+    it('should error when no entry point specified', async () => {
+        const expectedError = {
+            status: 'error',
+            message: 'Error with config. Directory: undefined, Entry: testDir'
+        };
+        const config = {
+            entry: 'testDir'
+        };
+        await expect(skeletorLocalServer().run(config, options)).rejects.toEqual(expectedError);
+    });
+
+    it('should spin up a local server', () => {
+        const expectedResp = {
+            status: 'running'
+        };
+        const expectedMessage = 'Started server on port 3000';
+        const config = {
+            entry: 'testDir',
+            currentDirectory: 'testDir'
+        };
+
+        return skeletorLocalServer().run(config, options).then(resp => {
+            expect(consoleSpy).toHaveBeenCalledTimes(1);
+            expect(consoleSpy).toHaveBeenCalledWith(expectedMessage);
+            expect(resp).toEqual(expectedResp);
```

Comment:
It looks like this test is just testing that your console.log() is firing after the server started. It might be helpful if this test tested that the correct express methods (use() and listen()) were called with the correct parameters.